spec/controllers/users_controller_spec.rb

```diff
@@ -32,59 +32,52 @@
         email: 'newemail@gmail.com',
         language: 'gl'
       }
-      user = create(:user)
       patch :update, params: { id: user.id, user: updated_params }
-      user.reload
       expect(response).to have_http_status(:ok)
-      expect(user.name).to eq(updated_params[:name])
-      expect(user.email).to eq(updated_params[:email])
-      expect(user.language).to eq(updated_params[:language])
+      expect(user.reload.name).to eq(updated_params[:name])
+      expect(user.reload.email).to eq(updated_params[:email])
+      expect(user.reload.language).to eq(updated_params[:language])
     end
 
     it 'returns an error if the user update fails' do
-      user = create(:user)
       patch :update, params: { id: user.id, user: { name: nil } }
       expect(response).to have_http_status(:bad_request)
       expect(user.reload.name).to eq(user.name)
     end
 
     it 'updates the avatar' do
-      user = create(:user)
       patch :update, params: { id: user.id, user: { avatar: fixture_file_upload(file_fixture('default-avatar.png'), 'image/png') } }
       expect(user.reload.avatar).to be_attached
     end
 
     it 'deletes the avatar' do
-      user = create(:user)
       user.avatar.attach(io: fixture_file_upload('default-avatar.png'), filename: 'default-avatar.png', content_type: 'image/png')
       expect(user.reload.avatar).to be_attached
       delete :purge_avatar, params: { id: user.id }
       expect(user.reload.avatar).not_to be_attached
     end
 
     it 'returns an error if the avatar is a pdf' do
-      user = create(:user)
       patch :update, params: { id: user.id, user: { avatar: fixture_file_upload(file_fixture('default-pdf.pdf'), 'pdf') } }
       expect(user.reload.avatar).not_to be_attached
     end
 
     it 'returns an error if the avatar size is too large' do
-      user = create(:user)
       patch :update, params: { id: user.id, user: { avatar: fixture_file_upload(file_fixture('large-avatar.jpg'), 'jpg') } }
       expect(user.reload.avatar).not_to be_attached
     end
   end
 
   describe 'DELETE users#destroy' do
     it 'deletes the user' do
-      user = create(:user)
       expect(response).to have_http_status(:ok)
       expect { delete :destroy, params: { id: user.id } }.to change(User, :count).by(-1)
     end
 
     it 'does not delete any user if the user id is invalid' do
       expect { delete :destroy, params: { id: 'invalid-id' } }.not_to change(User, :count)
-      expect(response).to have_http_status(:not_found)
+      # TODO - maybe ensure_authorized should return a :not_found if the user is not authorized AND the param doesn't exists?
+      expect(response).to have_http_status(:unauthorized)
```

Comment:
Related to https://github.com/bigbluebutton/greenlight/pull/3729/files#r938277438